Dockerfile

```
@@ -7,5 +7,6 @@ COPY backup.sh /usr/local/bin/backup.sh
 
 RUN microdnf update -y && rm -rf /var/cache/yum
 RUN microdnf install findutils -y && microdnf clean all
+RUN curl -s https://dl.min.io/client/mc/release/linux-amd64/mc -o /usr/local/bin/mc && chmod +x /usr/local/bin/mc
```

given that we're on ubi9, wouldn't it be preferable to the cli from `https://dl.min.io/client/mc/release/linux-amd64/mc.rpm`?

---

```suggestion
RUN curl -O https://dl.min.io/client/mc/release/linux-amd64/mc.rpm \
 && rpm -ih mc.rpm \
 && rm mc.rpm
```

note: all references to `mc` would also need to be replaced with `mcli` because the name overlaps with the `mc` package from some RPM distros.

---

i can totally agree, thanks for the nice input, will apply that!